certified-connectors/Integration Toolbox/apiDefinition.swagger.json

```diff
@@ -246,9 +250,10 @@
     },
     "/pgpdecrypt": {
       "post": {
-        "operationId": "PGPDecrypt",
+        "operationId": "PGPDecrypt (deprecated)",
```

Comment:
@lfalck the operationId is changes here. I think you meant to change the summary instead of this?
It is not allowed to change the operationIds. Please revert this change.